www/api/modules/payments/gateways/stripe/domain/service/card/card_service.py

```diff
@@ -18,7 +18,7 @@ def pay(
             return error, None
 
         return self.card_repository.pay_subscription({
-            "amount": subscription_type.transaction_amount * 100,
+            "amount": subscription_type.price,
```

Comment:
You need to add back the 100 this is to save the price in cents this way we can avoid pointer issues